src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/services/service-daemon-list/service-daemon-list.component.html

```diff
@@ -23,9 +23,17 @@
           <cd-table *ngIf="hasOrchestrator"
                     #daemonsTable
                     [data]="daemons"
+                    selectionType="single"
                     [columns]="columns"
                     columnMode="flex"
-                    (fetchData)="getDaemons($event)">
+                    (fetchData)="getDaemons($event)"
+                    (updateSelection)="updateSelection($event)">
+            <cd-table-actions id="pool-list-actions"
+                              class="table-actions"
+                              [selection]="selection"
+                              [permission]="permissions.hosts"
```

Comment:
`permissions.hosts` doesn't look right here. Did you tried importing `Permission` instead of `Permissions` and using it here just as `permissions`

Reply:
should there be a daemon specific permission ? Or are permissions tied to ceph core?

Reply:
I am not sure if its tied up to ceph core. But I know there isn't any daemon specific permission out there. @epuertat @alfonsomthd tagging you guys for a better clarity here.

Reply:
The problem here is that this action apply to any kind of daemon (daemon-agnostic) so I see 2 options:

* We stick to `permissions.hosts` if we understand that this the minimum required as daemons are distributed along hosts.
* We create a new kind of permission similar to the existing management role `cluster-manager` that allows daemon-type-agnostic daemon management.

Reply:
as per our last meeting, `permission.hosts` it is.